site/content/usage/experimental/01-gitops.md

```diff
@@ -32,57 +32,82 @@ Or use a config file:
 EKSCTL_EXPERIMENTAL=true eksctl install flux -f examples/01-simple-cluster.yaml --git-url=git@github.com:weaveworks/cluster-1-gitops.git --git-email=johndoe+flux@weave.works
 ```
 
+Note that, by default, `eksctl install flux` installs [Helm](https://helm.sh/) server components to the cluster (it 
+installs [Tiller](https://helm.sh/docs/glossary/#tiller) and the [Flux Helm Operator](https://github.com/fluxcd/helm-operator)). To
+disable the installation of the Helm server components, pass the flag `--with-helm=false`.
+
 Full example:
 
 ```bash
```

Comment:
Daniel has been using "console" instead of bash. Check this example https://deploy-preview-1138--eksctl.netlify.com/gitops-quickstart/setup-gitops/

Reply:
sure